Automattic/wp-calypso
# Fix: Send Invitation button needs to be clicked twice when not focused

Closes: https://github.com/Automattic/wp-calypso/issues/78886

## Proposed Changes

This PR fixes the need for the `Send Invitation` button to be clicked twice when it is not focused on `Users -> Add New` tab. 

## Testing Instructions

* Navigate to `Users -> Add New` 
* Type a valid username or email in the field
* Click on `Send Invitation` 
* Confirm that the invitation gets sent on the first click

**Testing invalid username**

* Navigate to `Users -> Add New` 
* Type an invalid username or email (you can use `katerynakodonenk` which is invalid)
* Confirm that you see the error that the user is not found
* Confirm that you can't send an invitation

**Notes**

* The issue with the button needed to be clicked twice is caused by `onTokenBlur` function that would only started the validation of tokens when the user clicks outside of the input field. This would work okay if we had another field that was mandatory and that would make the user click out. Since this is not the case, it requires the user to click somewhere first before the validation can start and the button is working
* We moved with @kozer , the validation of tokens to `onChange` instead and used `useDebouncedCallback` to prevent validation requests to the API on every keystroke
## Pre-merge Checklist
